bin/helpers/helper.js

```diff
@@ -0,0 +1,292 @@
+
```

Comment:
Imo the name of this file should be changed to `pluginHelpers` or a name that described that these are custom helpers being used in the context of `a11y` and `o11y`. Or alternatively add a comment on the top that this file contains helper methods for `A11y` and `O11y`.

Reply:
Done